modules/test-jpa/src/main/java/com/devonfw/module/test/common/base/clean/TestCleanerPluginNone.java

```diff
@@ -8,7 +8,7 @@
 public class TestCleanerPluginNone implements TestCleanerPlugin {
 
   @Override
-  public void cleanup() {
+  public void cleanup() throws Exception {
```

Comment:
Do not do this. Please use RuntimeExceptions and avoid checked exceptions wherever possible:
https://github.com/devonfw/devon4j/blob/develop/documentation/guide-exceptions.asciidoc

Also we can not afterwards change APIs in an incompatible manner:
Existing projects would get compiler errors after upgrading devon4j when you do this change.